warehouse/query-core/src/main/java/datawave/query/transformer/FieldIndexCountQueryTransformer.java

```diff
@@ -18,17 +13,21 @@
 import datawave.webservice.query.logic.BaseQueryLogicTransformer;
 import datawave.webservice.query.result.event.EventBase;
 import datawave.webservice.query.result.event.FieldBase;
-import datawave.webservice.query.result.event.ResponseObjectFactory;
 import datawave.webservice.query.result.event.Metadata;
+import datawave.webservice.query.result.event.ResponseObjectFactory;
 import datawave.webservice.result.BaseQueryResponse;
 import datawave.webservice.result.EventQueryResponseBase;
-
 import org.apache.accumulo.core.data.Key;
 import org.apache.accumulo.core.data.Value;
 import org.apache.accumulo.core.security.Authorizations;
 import org.apache.hadoop.io.Text;
 import org.apache.log4j.Logger;
 
+import java.util.ArrayList;
```

Comment:
No changes needed to this class

Reply:
My point on this one is that the ONLY changes you have to this class are import reordering.  Changes to this class are completely irrelevant to the pull request and can be reverted.

Reply:
performed ` git checkout integration warehouse/query-core/src/main/java/datawave/query/transformer/FieldIndexCountQueryTransformer.java`